example/manage.py

```diff
@@ -6,7 +6,7 @@
 
 def main():
     """Main function for executing arguments."""
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "server.settings")
```

Comment:
What are these changes?

Reply:
I moved the settings file under `server` so I had to adjust the python path to it.